src/providers/flagsView.ts

```diff
@@ -1,364 +1,417 @@
 import * as vscode from 'vscode';
 import { FeatureFlag } from '../models';
 import { LaunchDarklyAPI } from '../api';
-import { Configuration, getIsTreeviewEnabled } from '../configuration';
+import { Configuration } from '../configuration';
 import { FlagStore } from '../flagStore';
 import * as path from 'path';
+import { debounce } from 'lodash';
 
-export class LaunchDarklyTreeViewProvider implements vscode.TreeDataProvider<FlagValue> {
+const COLLAPSED = vscode.TreeItemCollapsibleState.Collapsed;
+const NON_COLLAPSED = vscode.TreeItemCollapsibleState.None;
+
+export class LaunchDarklyTreeViewProvider implements vscode.TreeDataProvider<FlagNode> {
 	private readonly api: LaunchDarklyAPI;
 	private config: Configuration;
 	private flagStore: FlagStore;
-	private flagValues: Array<FlagValue>;
+	private flagNodes: Array<FlagNode>;
 	private ctx: vscode.ExtensionContext;
-	private _onDidChangeTreeData: vscode.EventEmitter<FlagValue | undefined> = new vscode.EventEmitter<
-		FlagValue | undefined
-	>();
-	readonly onDidChangeTreeData: vscode.Event<FlagValue | undefined> = this._onDidChangeTreeData.event;
+	private readonly streamingConfigOptions = ['accessToken', 'baseUri', 'streamUri', 'project', 'env'];
```

Comment:
Looks like we're defining this same array in multiple providers - maybe factor it out and export in `configuration.ts`?